docs/guide/from-2022a-to-2022b.md

```diff
@@ -0,0 +1,122 @@
+# Upgrade from R2022a to R2022b
+PROTO nodes are a powerful way of describing complex objects or robots.
+PROTO files however evolve, bugs are fixed, and given that there is no inherent way of defining a specific version of a PROTO to be used when loading a world, older simulations no longer behave the same way due to these changes.
+Moreover, with the addition of JavaScript as a template language for procedural PROTO nodes, having PROTO files as external assets that can be downloaded on the fly opens up, for example, the possibility of rendering PROTO files directly from the web.
+
+For these reasons, from R2022b, all PROTO files used by a world need to be declared using the `EXTERNPROTO` keyword.
+Similarly, every PROTO inserted during the execution of the simulation by a supervisor needs to be declared an `IMPORTABLE EXTERNPROTO` prior to import it.
+
+Changes breaking the backward compatibility are listed in bold in the [Change Log](../reference/changelog-r2022.md).
+
+## What is New
+### What is EXTERNPROTO
+
+It is a keyword at the top of your world/proto file that allows Webots to know where it should search for a defined proto. Most of the time it will be on the official github respository of Cyberbotics.
```

Comment:
```suggestion
It is a keyword at the top of your world/PROTO file that allows Webots to know where it should search for a defined PROTO. Most of the time it will be on the official GitHub respository of Cyberbotics.
```